packages/functional-tests/tests/react-conversion/signup.spec.ts

```diff
@@ -177,3 +189,131 @@ test.describe('severity-1 #smoke', () => {
     });
   });
 });
+
+test.describe('severity-2 #smoke', () => {
+  test.describe('signup react', () => {
+    test('signup invalid email', async ({ page, pages: { signupReact } }) => {
+      skipCleanup = true;
+      email = 'invalid';
+      await signupReact.goto();
+      await signupReact.fillOutEmailFirst(email);
+      await expect(
+        page.getByText('Valid email required', { exact: true })
+      ).toBeVisible();
+    });
+
+    test('empty email', async ({ page, pages: { signupReact } }) => {
+      skipCleanup = true;
+      await signupReact.goto();
+      await signupReact.fillOutEmailFirst('');
+      await expect(
+        page.getByText('Valid email required', { exact: true })
+      ).toBeVisible();
+    });
+
+    test('coppa is too young', async ({ page, pages: { signupReact } }) => {
+      skipCleanup = true;
+      await signupReact.goto();
+      await signupReact.fillOutEmailFirst(email);
+      await signupReact.fillOutSignupForm(PASSWORD, '12');
+      await page.waitForURL(/cannot_create_account/);
+    });
+
+    test('Visits the privacy policy links save information upon return', async ({
+      page,
+      pages: { signupReact },
+    }) => {
+      skipCleanup = true;
+      await signupReact.goto();
+      await signupReact.fillOutEmailFirst(email);
+      await signupReact.fillOutSignupForm(PASSWORD, '21', false);
+      await signupReact.visitPrivacyPolicyLink();
+      await page.waitForURL(/legal\/privacy/);
+      await page.goBack();
+
+      // TBD: Currently not supported. Do we need to support this?
+      // expect(await signupReact.getEmail().inputValue()).toEqual(email);
+      // expect(await signupReact.getPassword().inputValue).toEqual(PASSWORD);
+      // expect(await signupReact.getPasswordConfirm().inputValue).toEqual(PASSWORD);
+      // expect(await signupReact.getAge().inputValue).toEqual('21');
+    });
+
+    test('Visits the terms of service links save information upon return', async ({
+      page,
+      pages: { signupReact },
+    }) => {
+      skipCleanup = true;
+      await signupReact.goto();
+      await signupReact.fillOutEmailFirst(email);
+      await signupReact.fillOutSignupForm(PASSWORD, '21', false);
+      await signupReact.visitTermsOfServiceLink();
+      await page.waitForURL(/legal\/terms/);
+      await page.goBack();
+
+      // TBD: Currently not supported. Do we need to support this?
```

Comment:
I've noticed that clicking back after accessing the terms from backbone does not allow the user to go back (page reloads then fails to navigate back to signin/signup on second click of the back arrow - blank page is displayed). At least the React version allows to go back, even if the form is not filled. We might want to consider opening the terms in a new tab?

Reply:
I see that too, double-click sometimes works to go back, but not always.